build.fsx

```diff
@@ -214,6 +214,42 @@ Target "GitHubRelease" (fun _ ->
 )
 
 // --------------------------------------------------------------------------------------
+// .NET CLI and .NET Core
+
+let isDotnetCLIInstalled = Shell.Exec("dotnet", "--info") = 0
+let assertExitCodeZero x = if x = 0 then () else failwithf "Command failed with exit code %i" x
+
+Target "DotnetCliBuild" (fun _ ->
+    let fsLex  = @"lib/bootstrap/4.0/fslex.exe"
+    let fsYacc = @"lib/bootstrap/4.0/fsyacc.exe"
+    let outPath = @"src/fsharp/FSharp.Compiler.Service.netcore/"
+    let lexArgs = @" --lexlib Internal.Utilities.Text.Lexing"
+    let yaccArgs = @" --internal --parslib Internal.Utilities.Text.Parsing"
+    let module1 = @" --module Microsoft.FSharp.Compiler.AbstractIL.Internal.AsciiParser"
+    let module2 = @" --module Microsoft.FSharp.Compiler.Parser"
+    let module3 = @" --module Microsoft.FSharp.Compiler.PPParser"
+    let open1 = @" --open Microsoft.FSharp.Compiler.AbstractIL"
+    let open2 = @" --open Microsoft.FSharp.Compiler"
+    let open3 = @" --open Microsoft.FSharp.Compiler"
+    let gen1 = @" ../FSComp.txt ./FSComp.fs ./FSComp.resx"
+    let gen2 = @" ../fsi/FSIstrings.txt ./FSIstrings.fs ./FSIstrings.resx"
+    let options = " --configuration Release"
+    
+    Shell.Exec("dotnet", "restore", outPath) |> ignore //assertExitCodeZero
+
+    Shell.Exec("dotnet", "fssrgen" + gen1, outPath) |> assertExitCodeZero
+    Shell.Exec("dotnet", "fssrgen" + gen2, outPath) |> assertExitCodeZero
```

Comment:
maybe inline `gen1`, `gen2`, i think

```
Shell.Exec("dotnet", "fssrgen ../FSComp.txt ./FSComp.fs ./FSComp.resx", outPath) |> assertExitCodeZero
```

it's easier to read, instead of try find the variables